app/controllers/RecommendationController.php

```diff
@@ -34,6 +34,10 @@ public function create()
 
     $vars = (object) $this->settings->getSpecifiedSettingsVars(['recommendation_create_help_text']);
 
+    if(!Auth::check()){
```

Comment:
let's try this with a filter https://github.com/DoSomething/longshot/blob/master/app/controllers/RecommendationController.php#L12